packages/react-icons/src/components.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as icons from '@fluentui/react-icons';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ReactIs from 'react-is';
+
+const iconsDir = 'src/components';
+
+describe('Icon components', () => {
+  fs.readdirSync(iconsDir).forEach(file => {
+    const componentName = path.basename(file, '.tsx');
+    // tslint:disable-next-line:no-any
+    const IconComponent = (icons as any)[componentName];
+
+    test(`${componentName} has 'Icon' suffix in the name`, () => {
+      expect(componentName.endsWith('Icon')).toBeTruthy();
```

Comment:
If this returns a `boolean` just check if it's `true`. Probably not hugely important, but it's more robust.